mix.exs

```diff
@@ -11,7 +11,7 @@ defmodule SweetXml.Mixfile do
     [
       app: app(),
       version: version(),
-      elixir: "~> 1.0",
+      elixir: "~> 1.11",
```

Comment:
I've updated our requirements in the organization profile and contributing.md
Can you bump this to 1.12 ?
```suggestion
      elixir: "~> 1.12",
```